index.html

```diff
@@ -485,6 +489,20 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 <a data-for="PerformanceResourceTiming">timing info</a>'s
 [=fetch timing info/end time=] and the <a>relevant global object</a> for <a>this</a>.
 See [=/fetch=] for more info.
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>encodedBodySize</dfn> getter
+steps are to return <a>this</a>'s
+<a data-for="PerformanceResourceTiming">timing info</a>'s [=fetch timing info/encoded body size=].
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>decodedBodySize</dfn> getter
+steps are to return <a>this</a>'s
+<a data-for="PerformanceResourceTiming">timing info</a>'s [=fetch timing info/decoded body size=].
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>transferSize</dfn> getter
+steps are to perform the following steps:
+<ol>
+<li><p>If <a>this</a>'s <a data-for="PerformanceResourceTiming">cache mode</a> is
+"<code>local</code>", then return 0.
+<li><p>Return <a>this</a>'s
+<a data-for="PerformanceResourceTiming">timing info</a>'s [=fetch timing info/encoded body size=].
```

Comment:
If we add a fixed size here for the headers, we can close https://github.com/w3c/resource-timing/issues/238#issuecomment-762297269

Reply:
Sure, I can do that, what value should I add?